app/screens/send-bitcoin-screen/scanning-qrcode-screen.tsx

```diff
@@ -62,6 +64,31 @@ const styles = EStyleSheet.create({
   },
 })
 
+const getBillpayData = (client) => {
+  const baseUrl = "https://billpay.mainnet.galoy.io"
+  const fullUrl = `${baseUrl}/?client=${client}`
+  return fetch(fullUrl)
+    .then((response) => response.json())
+    .then((data) => {
+      if (data.status === "ERROR") {
+        throw new Error("Error while requesting billpay data")
+      }
+
+      const urlFormat = {
+        protocol: "https",
+        host: LNPAY_DOMAIN,
+        pathname: data.username,
+        query: {
+          amount: data.balance,
+          memo: `billpay ${data.customer}-${client}`,
+          currency: data.currency,
+        },
+      }
+
+      return url.format(urlFormat)
```

Comment:
It seems a little clunky to format all the information into a url string and then decode it again down in the parsing function without using it to make a network request. Would passing the data in a typed object instead of a string be a worthwhile refactoring in the future? It makes sense to leave it as a string for now because it's how all the functions on the send bitcoin screen are set up.